aria-hidden not focusable, add hidden svg in link example (#1933)
auto-wcag/auto-wcag: _rules/aria-hidden-no-focusable-content-6cfa84.md

</div>
```

#### Passed Example 6

> This `svg` element with `aria-hidden` is a descendant of a focusable link, rather than an ancestor.

Reviewer: ```suggestion
This `svg` element with `aria-hidden` does not have a focusable descendant, focusable ancestors are not a problem for this rule.
```
I usually prefer description that follow the wording of expectation.
